### ISSUE TITLE
qhull package doesn't contains static library anymore

### ISSUE THREAD

**Issue by jorisv** [conda-forge/qhull-feedstock#18]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

qhull package use to contains qhull static library (libqhullcpp.a, libqhullstatic.a and libqhullstatic_r.a).

[Last PR](https://github.com/conda-forge/qhull-feedstock/pull/17) remove the [static build option](https://github.com/conda-forge/qhull-feedstock/pull/17/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR8).

This create an issue on some packet I manage ([pinocchio](https://github.com/conda-forge/pinocchio-feedstock) and [hpp-fcl](https://github.com/conda-forge/hpp-fcl-feedstock/)) and block the release of [hpp-pinocchio](https://github.com/humanoid-path-planner/hpp-pinocchio/issues/205).

Is there some reason to remove static library from the package ?

### Installed packages

```shell
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
ampl-mp                   3.1.0             h2cc385e_1006    conda-forge
assimp                    5.4.1                h8343317_0    conda-forge
bzip2                     1.0.8                hd590300_5    conda-forge
c-ares                    1.28.1               hd590300_0    conda-forge
ca-certificates           2024.7.4             hbcca054_0    conda-forge
casadi                    3.6.5           py312h1a19fd5_4    conda-forge
cmake                     3.30.0               hf8c4bd3_0    conda-forge
console_bridge            1.0.2                h924138e_1    conda-forge
eigen                     3.4.0                h00ab1b0_0    conda-forge
eigenpy                   3.6.0           py312hcdefc7e_0    conda-forge
example-robot-data        4.1.0              pyh7edd284_3    conda-forge
hpp-environments          5.1.0           py312h3402730_2    conda-forge
hpp-fcl                   2.4.4           py312hf2ad3da_5    conda-forge
hpp-util                  5.1.0                h41be2f8_0    conda-forge
icu                       73.2                 h59595ed_0    conda-forge
ipopt                     3.14.16              h3696c94_4    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
krb5                      1.21.3               h659f571_0    conda-forge
ld_impl_linux-64          2.40                 hf3520f5_7    conda-forge
libblas                   3.9.0           22_linux64_openblas    conda-forge
libboost                  1.84.0               hba137d9_3    conda-forge
libboost-devel            1.84.0               h00ab1b0_3    conda-forge
libboost-headers          1.84.0               ha770c72_3    conda-forge
libboost-python           1.84.0          py312h389efb2_3    conda-forge
libboost-python-devel     1.84.0          py312h9cebb41_3    conda-forge
libcblas                  3.9.0           22_linux64_openblas    conda-forge
libcurl                   8.8.0                hca28451_1    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 hd590300_2    conda-forge
libexpat                  2.6.2                h59595ed_0    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 14.1.0               h77fa898_0    conda-forge
libgfortran-ng            14.1.0               h69a702a_0    conda-forge
libgfortran5              14.1.0               hc5f4f2c_0    conda-forge
libgomp                   14.1.0               h77fa898_0    conda-forge
libhwloc                  2.11.0          default_h5622ce7_1000    conda-forge
libiconv                  1.17                 hd590300_2    conda-forge
liblapack                 3.9.0           22_linux64_openblas    conda-forge
libnghttp2                1.58.0               h47da74e_1    conda-forge
libnsl                    2.0.1                hd590300_0    conda-forge
libopenblas               0.3.27          pthreads_hac2b453_1    conda-forge
libosqp                   0.6.3                h59595ed_0    conda-forge
libqdldl                  0.1.5                h27087fc_1    conda-forge
libscotch                 7.0.4                h2fe6a88_5    conda-forge
libspral                  2024.05.08           hdf4021c_2    conda-forge
libsqlite                 3.46.0               hde9e2c9_0    conda-forge
libssh2                   1.11.0               h0841786_0    conda-forge
libstdcxx-ng              14.1.0               hc0a3c3a_0    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libuv                     1.48.0               hd590300_0    conda-forge
libxcrypt                 4.4.36               hd590300_1    conda-forge
libxml2                   2.12.7               hc051c1a_1    conda-forge
libzlib                   1.3.1                h4ab18f5_1    conda-forge
metis                     5.1.0             h59595ed_1007    conda-forge
mumps-include             5.7.2                ha770c72_0    conda-forge
mumps-seq                 5.7.2                h6e8dedb_0    conda-forge
ncurses                   6.5                  h59595ed_0    conda-forge
numpy                     1.26.4          py312heda63a1_0    conda-forge
octomap                   1.9.8                h924138e_0    conda-forge
openssl                   3.3.1                h4ab18f5_1    conda-forge
pinocchio                 3.1.0           py312he572396_0    conda-forge
pkg-config                0.29.2            h36c2ea0_1008    conda-forge
proxsuite                 0.6.6           py312h2492b07_0    conda-forge
python                    3.12.4          h194c7f8_0_cpython    conda-forge
python_abi                3.12                    4_cp312    conda-forge
qhull                     2020.2               h434a139_3    conda-forge
readline                  8.2                  h8228510_1    conda-forge
rhash                     1.4.4                hd590300_0    conda-forge
scipy                     1.14.0          py312hc2bc53b_1    conda-forge
simde                     0.8.2                h297d8ca_0    conda-forge
tinyxml                   2.6.2                h4bd325d_2    conda-forge
tinyxml2                  10.0.0               h59595ed_0    conda-forge
tk                        8.6.13          noxft_h4845f30_101    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
unixodbc                  2.3.12               h661eb56_0    conda-forge
urdfdom                   4.0.0                hee28ff1_1    conda-forge
urdfdom_headers           1.1.1                h00ab1b0_0    conda-forge
xz                        5.2.6                h166bdaf_0    conda-forge
zlib                      1.3.1                h4ab18f5_1    conda-forge
zstd                      1.5.6                ha6fb4c9_0    conda-forge
```

### Environment info

```shell
active environment : /tmp/test/.pixi/envs/default
    active env location : /tmp/test/.pixi/envs/default
            shell level : 2
       user config file : /home/jvaillan/.condarc
 populated config files : /home/jvaillan/.condarc
          conda version : 24.3.0
    conda-build version : not installed
         python version : 3.11.5.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=skylake
                          __conda=24.3.0=0
                          __cuda=12.2=0
                          __glibc=2.35=0
                          __linux=6.5.0=0
                          __unix=0=0
       base environment : /home/jvaillan/miniconda3  (writable)
      conda av data dir : /home/jvaillan/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/jvaillan/miniconda3/pkgs
                          /home/jvaillan/.conda/pkgs
       envs directories : /home/jvaillan/miniconda3/envs
                          /home/jvaillan/.conda/envs
               platform : linux-64
             user-agent : conda/24.3.0 requests/2.31.0 CPython/3.11.5 Linux/6.5.0-41-generic ubuntu/22.04.4 glibc/2.35 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.3
                UID:GID : 673165:21076
             netrc file : None
           offline mode : False
```

**Comment by ymontmarin** [comment]
This issue is preventing this PR on `hpp-pinocchio` feedstock: https://github.com/conda-forge/hpp-pinocchio-feedstock/pull/29

**Comment by xhochy** [comment]
Fixed in #19
